float64 temperature
float64 bus_voltage

float64 output_percent
float64 output_voltage
float64 output_current

float64 position
float64 velocity

bool fwd_limit
bool rev_limit